<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>notesensesetting</class>
 <widget class="QDialog" name="notesensesetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #3E2B1F; /* 深棕色背景 */

</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>330</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxNoteCount">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
;</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>5</number>
   </property>
   <property name="stepType">
    <enum>QAbstractSpinBox::StepType::AdaptiveDecimalStepType</enum>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="notesettinglabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;</string>
   </property>
   <property name="text">
    <string>Note Game Setting</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="notenumlable">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Myanmar Text</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::ContextMenuPolicy::DefaultContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>每次测试音符数量</string>
   </property>
   <property name="textFormat">
    <enum>Qt::TextFormat::MarkdownText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="speedlable">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Myanmar Text</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::ContextMenuPolicy::DefaultContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>音符播放时间间隔</string>
   </property>
   <property name="textFormat">
    <enum>Qt::TextFormat::MarkdownText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxSpeed">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>160</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
;</string>
   </property>
   <property name="suffix">
    <string>ms</string>
   </property>
   <property name="minimum">
    <number>300</number>
   </property>
   <property name="maximum">
    <number>1500</number>
   </property>
   <property name="stepType">
    <enum>QAbstractSpinBox::StepType::AdaptiveDecimalStepType</enum>
   </property>
   <property name="value">
    <number>500</number>
   </property>
  </widget>
  <widget class="QLabel" name="scalelable">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>100</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Myanmar Text</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::ContextMenuPolicy::DefaultContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>音阶选择（可多选）</string>
   </property>
   <property name="textFormat">
    <enum>Qt::TextFormat::MarkdownText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetScales">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>160</y>
     <width>191</width>
     <height>91</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SelectionMode::ExtendedSelection</enum>
   </property>
   <item>
    <property name="text">
     <string>1(A0-B1)</string>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2(C2-B3)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3(C3-B4)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4(C4-B5)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5(C5-B6)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6(C6-B7)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7(C7-C8)</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>notesensesetting</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>notesensesetting</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
